include/wil/filesystem.h

```diff
@@ -624,10 +624,15 @@ namespace wil
 
                     if (m_folderHandle)
                     {
-                        CancelIoEx(m_folderHandle.get(), &m_overlapped);
-
-                        DWORD bytesTransferredIgnored = 0;
-                        GetOverlappedResult(m_folderHandle.get(), &m_overlapped, &bytesTransferredIgnored, TRUE);
+                       BOOL cancelRes = CancelIoEx(m_folderHandle.get(), &m_overlapped);

```

Comment:
nit, the indentation here looks wrong. 4 spaces, no tabs please.

Reply:
fixed. Don't know how I managed to add 3 spaces instead of 4.